### PR TITLE
Merge Master into Dev Working Branch

### PR DESCRIPTION
**Pull request by StephenHodgson**
See #1614

### CODE REVIEW

File: Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs [hunk 1]
```diff
@@ -191,10 +191,6 @@ public void CalculateBoundaryVolume()
                     }
                 }
             }
-            else
```
Comment (keveleigh) [inline Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: Looks like this is removing something that was added in the Dev branch.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: Do we really need a log that this always returns false when it's stated in the comments above?

Comment (StephenHodgson) [reply Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: This is kinda strange, I did a straight merge from the master branch, without any edits.

Comment (keveleigh) [reply Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: I think this was added by @NeerajW as a way to validate when that API becomes fixed on Unity's end.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: Idk, having optional debug logs like this is probably useful.  IMO, Production apps wouldn't want this kind of thing spamming the console.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs]: Not to mention the performance impact debug statements have in general.

**Pull request merged**